.bcr/source.template.json

```diff
@@ -0,0 +1,5 @@
+{
+    "url": "https://github.com/realm/SwiftLint/releases/download/{TAG}/bazel.tar.gz",
```

Comment:
I assume the bazel archive is the right one?

Reply:
Yes